include/common_structs.h

```diff
@@ -1465,8 +1476,8 @@ typedef struct DecorationTable {
     /* 0x8B8 */ s8 decorationType[2];
     /* 0x8BA */ u8 unk_8BA[2];
     /* 0x8BC */ u8 unk_8BC[2];
-    /* 0x8C0 */ s16 unk_8C0[6];
-    /* 0x8BE */ char unk_8BE[30];
+    /* 0x8C0 */ s16 unk_8C0[4];
+    /* 0x8BE */ decorationUnk unk_8C6[2];
```

Comment:
This field should then be `unk_8C6` (which you named correctly) but please also edit the offset-comment!